babel/numbers.py

```diff
@@ -448,7 +448,7 @@ def format_currency(
     >>> format_currency(1099.98, 'JPY', locale='en_US')
     u'\\xa51,100'
     >>> format_currency(1099.98, 'COP', u'#,##0.00', locale='es_ES')
-    u'1.100'
+    u'1.099,98'
```

Comment:
This looked a little suspect at first, but seems to be legit:

* https://www.unicode.org/cldr/charts/34/delta/supplemental-data.html#COP
* https://unicode.org/cldr/trac/ticket/10788

[COP (the Colombian peso)](https://en.wikipedia.org/wiki/Colombian_peso) doesn't have cash money smaller than 1 peso, but for accounting, etc. purposes, one peso is still 100 centimos.